*** QuantLib-0.3.0-old/config/ltmain.sh	Wed Jan 16 02:16:27 2002
--- QuantLib-0.3.0/config/ltmain.sh	Fri Sep 13 03:35:58 2002
***************
*** 2931,2937 ****
--- 2931,2948 ----
  	if test -n "$export_symbols" && test -n "$archive_expsym_cmds"; then
  	  eval cmds=\"$archive_expsym_cmds\"
  	else
+           save_deplibs="$deplibs"
+           for conv in $convenience; do
+             tmp_deplibs=
+             for test_deplib in $deplibs; do
+               if test "$test_deplib" != "$conv"; then
+                 tmp_deplibs="$tmp_deplibs $test_deplib"
+               fi
+             done
+             deplibs="$tmp_deplibs"
+           done
  	  eval cmds=\"$archive_cmds\"
+           deplibs="$save_deplibs"
  	fi
  	save_ifs="$IFS"; IFS='~'
  	for cmd in $cmds; do
